ix-dev/community/windmill/templates/test_values/basic-values.yaml

```diff
@@ -0,0 +1,70 @@
+windmill:
+  postgres_image_selector: postgres_16_image
+  web_host: localhost
+  db_password: changeme
+  base_url: ":80"
+  address: ""
+  log_max_size: "20m"
+  log_max_file: "10"
```

Comment:
please use numbers here, (and on questions) and append the `m` on the template

Reply:
no longer needed